BootloaderCorePkg/Stage1A/Stage1A.c

```diff
@@ -565,7 +565,13 @@ ContinueFunc (
   // Print FSP version
   FspInfoHdr = (FSP_INFO_HEADER *)(UINTN)(PcdGet32 (PcdFSPTBase) + FSP_INFO_HEADER_OFF);
   CopyMem (ImageId, &FspInfoHdr->ImageId, sizeof (UINT64));
-  DEBUG ((DEBUG_INFO, "FSPV: ID(%a) REV(%08X)\n", ImageId, FspInfoHdr->ImageRevision));
+  if (FspInfoHdr->HeaderRevision <= 5) {
+    DEBUG ((DEBUG_INFO, "FSPV: ID(%a) REV(%08X)\n", ImageId, FspInfoHdr->ImageRevision));
+  } else {
+    DEBUG ((DEBUG_INFO, "FSPV: ID(%a) REV(%04X%04X%04X)\n", ImageId, ((FspInfoHdr->ImageRevision & 0xFFFF0000) >> 16),
+                                                                     ((FspInfoHdr->ExtendedImageRevision & 0xFF00) | ((FspInfoHdr->ImageRevision & 0xFF00) >> 8)),
+                                                                     (((FspInfoHdr->ExtendedImageRevision & 0x00FF) << 8) | (FspInfoHdr->ImageRevision & 0x00FF))));
```

Comment:
The code logic looks good to me
It is not necessary to have so many spaces in the line 572/573 to make the line quite long (>150?).
Prefer line char number no more than 120 if possible.